packages/node/src/notifier.js

```diff
@@ -63,12 +60,6 @@ const Bugsnag = {
 
     bugsnag._logger.debug('Loaded!')
 
-    bugsnag.leaveBreadcrumb = function () {
-      bugsnag._logger.warn('Breadcrumbs are not supported in Node.js yet')
-    }
-
-    bugsnag._config.enabledBreadcrumbTypes = []
```

Comment:
remove the patches that were preventing breadcrumbs from being used with the node notifier